src/main/java/frc/robot/commands/MoveDistance.java

```diff
@@ -51,10 +51,6 @@ public boolean isFinished() {
     double[] position = this.drivetrain.getPosition();
     if (position[0] == position[1])
       currentPosition = position[0];
-    if (Math.abs(currentPosition - targetDistance) == allowedError) {
-      return true;
-    } else {
-      return false;
-    }
+    return Math.abs(currentPosition - targetDistance) >= allowedError;
```

Comment:
This will return true if the error is greater than the allowed error... In other words, if the error is greater than the allowed, it will finish.